deploy/demo-cli/blox-create-deployment.py

```diff
@@ -28,7 +28,11 @@ def run_apigateway(params):
 	command = ["cloudformation", "describe-stack-resource", "--stack-name", params.stack, "--logical-resource-id", "ApiResource"]
 	restResource = common.run_shell_command(params.region, command)
 
-	command = ["apigateway", "test-invoke-method", "--rest-api-id", restApi['StackResourceDetail']['PhysicalResourceId'], "--resource-id", restResource['StackResourceDetail']['PhysicalResourceId'], "--http-method", "POST", "--headers", "{}", "--path-with-query-string", "/v1/environments/%s/deployments?deploymentToken=%s" % (params.environment, params.token), "--body", ""]
+	uri = '/v1/environments/%s/deployments' % params.environment
```

Comment:
We are sticking with v1 for now. I think we decided that the v1 APIs are going to be mutable until probably after v0.3.0.